### ISSUE TITLE
Can't create nested scopes (auto-scope branch)

### ISSUE THREAD

**Issue by naruaway** [cyclejs/cycle-http-driver#27]
In the code below, in ChildComponent, sources.HTTP will not be properly scoped.
Because scoped ```response$$``` does not have ```response$$.isolateSource``` or  ```response$$.isolateSink```.
```javascript
function ParentComponent(sources) {
  // do some works
  const childComponent = isolate(ChildComponent)(sources);
  // create sinks
  sinks.HTTP = Observable.merge(childComponent.HTTP, ...)
  return sinks;
}
isolate(ParentComponent)(sources);
```

**Comment by naruaway** [comment]
I think this commit can fix the problem.
May I create a pull request?
https://github.com/naruaway/cycle-http-driver/commit/2423c0e117e5f616452a6d2902394bfbefc6de3d

**Comment by staltz** [comment]
Good catch! Yes, please submit a PR

**Comment by naruaway** [comment]
Thank you for you response. I submitted a PR.
https://github.com/cyclejs/cycle-http-driver/pull/28

**Comment by staltz** [comment]
Published version 7.0.0-rc2!
